serverless/discover/knative-event-sources.adoc

```diff
@@ -0,0 +1,21 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="knative-event-sources"]
+= Event sources
+include::modules/common-attributes.adoc[]
+:context: knative-event-sources
+
+toc::[]
+
+A Knative _event source_ can be any Kubernetes object that generates or imports cloud events, and relays those events to another endpoint, known as a xref:../../serverless/develop/serverless-event-sinks.adoc#serverless-event-sinks[_sink_]. Sourcing events is critical to developing a distributed system that reacts to events.
```

Comment:
```suggestion
A Knative _event source_ can be any Kubernetes object that generates or imports cloud events and relays those events to another endpoint. The endpoint is known as a xref:../../serverless/develop/serverless-event-sinks.adoc#serverless-event-sinks[_sink_]. Sourcing events is critical to developing a distributed system that reacts to events.
```
Consider rephrasing to keep the sentence under 25 words (though this rephrasing might be a little awkward).

Reply:
I think it reads better as it is, but thanks for the suggestions :)